<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="radtype_a">
     <property name="text">
      <string>Type 'A'</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="radtype_b">
     <property name="text">
      <string>Type 'B'</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="radtype_c">
     <property name="text">
      <string>Type 'C'</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="typeInput">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_4"/>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Function end time (t_end)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Coefficient 'c1'</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="Gui::InputField" name="FuncACoefC1"/>
       </item>
       <item row="3" column="2">
        <widget class="Gui::InputField" name="FuncACoefC2"/>
       </item>
       <item row="4" column="2">
        <widget class="Gui::InputField" name="FuncACoefC3"/>
       </item>
       <item row="1" column="2">
        <widget class="Gui::InputField" name="tEndFuncA"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Coefficient 'c3'</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Specify the inputs for Type 'A'</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Coefficient 'c2'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Function value at t_start </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Function start time (t_start)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Function value at t_end s</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Function end time (t_end)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Gui::InputField" name="tStartFuncB"/>
       </item>
       <item row="2" column="1">
        <widget class="Gui::InputField" name="tEndFuncB"/>
       </item>
       <item row="3" column="1">
        <widget class="Gui::InputField" name="startValueFuncB"/>
       </item>
       <item row="4" column="1">
        <widget class="Gui::InputField" name="endValueFuncB"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Specify the inputs for Type 'B'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Function VALUE at t_start </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Function start time (t_start)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Function end time (t_end)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Function DERIVATIVE at t_end</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Gui::InputField" name="tStartFuncC"/>
       </item>
       <item row="2" column="1">
        <widget class="Gui::InputField" name="tEndFuncC"/>
       </item>
       <item row="3" column="1">
        <widget class="Gui::InputField" name="startValueFuncC"/>
       </item>
       <item row="4" column="1">
        <widget class="Gui::InputField" name="endDerivativeFuncC"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Specify the inputs for Type 'C'</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>gui:inputfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
